pom.xml

```diff
@@ -2257,6 +2257,7 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
+      <scope>test</scope>
```

Comment:
previously this caused issues with our assembly, because the ITs relied on it in order to run against a live cluster or standalone instance. please make sure that still works.

Reply:
Ok, I will check it. It is definitely not a good idea to include junit to the compile classpath. It also caused some issues when shading testing-util. Do you remember how can I reproduce the issue?

Reply:
IIRC it was as simple as building an assembly, running a psuedo distributed cluster out of it, and then trying to run the ITs that ship in the assembly against the cluster. e.g. ITBLL

Reply:
Thanks, I wasn't able to run IntegrationTestsDriver. It is fixed now.